crates/sp-domains/src/fraud_proof.rs

```diff
@@ -472,6 +472,8 @@ pub struct InvalidExtrinsicsRootProof {
     pub valid_bundle_digests: Vec<ValidBundleDigest>,
     /// Randomness Storage proof
     pub randomness_proof: StorageProof,
+    /// Timestamp storage proof
```

Comment:
Minor thing, but why is it "Storage" for randomness and "storage" for timestamp? Pick one and stick to it.